pkg/web/src/pages/Settings/cluster/EditClusterModal.tsx

```diff
@@ -85,14 +85,24 @@ export const EditClusterModal = React.memo(() => {
   const validateCraneUrl = (id: string) => {
     const res = { error: false, msg: '' };
     const cluster = clusters.find((cluster: any) => cluster.id === id);
-
+    let reg = /^(?=^.{3,255}$)(http(s)?:\/\/)?(www\.)?[a-zA-Z0-9][-a-zA-Z0-9]{0,62}(\.[a-zA-Z0-9][-a-zA-Z0-9]{0,62})+(:\d+)*(\/\w+\.\w+)*$/;
```

Comment:
The regex should cover more cases.

<img width="317" alt="image" src="https://github.com/gocrane/crane/assets/35299017/8e569c6c-82e7-4c2c-9e36-aae3e4a73321">

Reply:
Do you use this regex for validation? The problem here is' substring(0, craneUrl.lastIndexOf('/')) '